tools/docker/old-env/Dockerfile

```diff
@@ -19,17 +19,17 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y -q --no-install-recommends
 	apt-get clean autoclean && \
 	rm -rf /var/lib/apt/lists/{apt,dpkg,cache,log} /tmp/* /var/tmp/*
 
-RUN curl https://dl.google.com/go/go1.14.2.linux-amd64.tar.gz | tar -C /usr/local -xz
+RUN curl https://dl.google.com/go/go1.16.13.linux-amd64.tar.gz | tar -C /usr/local -xz
```

Comment:
You used "go1.17.6.linux-amd64.tar.gz" in the "docs/linux/setup.md".
Any reason to use the different one here?

Reply:
Yes, the "env" container tests the later versions, while the "old" container tests the earlier supported versions. We run tests with both containers on CI.